explorer/websockethandlers.go

```diff
@@ -21,6 +21,49 @@ import (
 // close an already closed connection.
 var ErrWsClosed = "use of closed network connection"
 
+func trimTxInfo(txs []*TxInfo) []*TrimmedTxInfo {
+	trimmedTxs := make([]*TrimmedTxInfo, 0, len(txs))
+	for _, tx := range txs {
+		voteValid := false
+		if tx.IsVote() {
+			voteValid = tx.VoteInfo.Validation.Validity
+		}
+		trimmedTx := &TrimmedTxInfo{
+			VinCount:  len(tx.Vin),
+			VoutCount: len(tx.Vout),
+			VoteValid: voteValid,
+			TxID:      tx.TxID,
+			Total:     tx.Total,
+			Fees:      tx.Fee.ToCoin(),
+			Coinbase:  tx.Coinbase,
+		}
+		trimmedTxs = append(trimmedTxs, trimmedTx)
+	}
+	return trimmedTxs
+}
+func trimBlockInfo(block *BlockInfo) *TrimmedBlockInfo {
+	// check tx info
+	txCount := len(block.Tx)
+	transactions := make([]*TxInfo, 0, txCount)
+	for _, tx := range block.Tx {
+		if !tx.Coinbase {
```

Comment:
Why does it exclude coinbase?  I think this function needs docs.

Reply:
I'm not sure why. I adapted the pattern I saw in the block page. When looping through the transactions in a block to display in the transactions table of the block page, the code excluded coinbase transactions. (`lines 385 to 400 of /views/block.tmpl`)

I just borrowed a leaf with the impression that it was normal.